src/frontend/web_application/src/modules/message/actions/requestMessages.js

```diff
@@ -0,0 +1,6 @@
+import { requestMessages as requestMessagesBase } from '../../../store/modules/message';
+import { tryCatchAxiosAction } from '../../../services/api-client';
+
+// @deprecated: use fetchMessages instead or request collection directly
```

Comment:
Why not refactor requestMessages directly ? Does it break many things ?

Reply:
yes :)